network-proto/node.proto

```diff
@@ -10,13 +10,12 @@ message TipRequest {}
 
 // Response message for method Tip.
 message TipResponse {
-    // Block identifier of the chain tip.
-    bytes id = 1;
-    // Block date of the chain tip.
-    string blockdate = 2;
+    // Serialized content of the tip block header.
+    bytes blockheader = 1;
 }
 
 // Request message for methods GetBlocks and GetHeaders.
+// (reserved for the future work)
```

Comment:
This message is perhaps not needed and should be removed.
We can restore the fields from git history when about to introduce the new random access method `GetChain`.